### ISSUE TITLE
Player quotes are muted

### ISSUE THREAD

**Issue by Artur16211** [momo5502/boiii#235]
### Bug Type

Dedicated Server

### BOIII Version

v0.0.1 (Latest)

### Environment (Operating System)

Microsoft Windows 11 (64-Bit)

### Reproducibility Rate

High: The bug can (almost) always be reproduced

### Steps to Reproduce

1. join on a Dedicated server.

### Expected Result

Characters usually speak when performing some action from killing zombies, running out of bullets, out of points, etc.

### Actual Result

Characters only speak when it is an action related to an easter egg, or a specific dialogue of a round.

In maps like Shi no numa or Verruck they are totally silenced.

### Severity/Priority

Minor

**Comment by diamante0018** [comment]
Yeah plutonium fixed it by dumping the sound aliases then modifying the Dedi to use the dumped aliases

**Comment by Artur16211** [comment]
Well, let's hope this solution will be applied to boiii

**Comment by diamante0018** [comment]
Can't repro.

**Comment by MotoLegacy** [comment]
there has not been a commit that identifies this was fixed, and i had shown it being an issue on the referenced issue in these logs. i will check this myself to make sure and send video shortly.

**Comment by MotoLegacy** [comment]
Still happening on Dedicated Servers. Note how neither the Nuke or Max Ammo Power-Ups triggered VOX playback but the scripted main quest VOX played at end of round correctly. Could it be you were testing on a server with someone fixing the sound alias manually?


https://github.com/momo5502/boiii/assets/26030250/17e24c04-c4e3-4473-9fb8-afb78ec24f1b
